tests/test_models/test_mutables/test_diffop.py

```diff
@@ -40,15 +41,16 @@ def test_forward_arch_param(self):
         op = MODELS.build(op_cfg)
         input = torch.randn(4, 32, 64, 64)
 
-        arch_param = op.build_arch_param()
+        diffmutator = DiffModuleMutator()
+        diffmutator.prepare_from_supernet(op)
```

Comment:
Use user-defined instead  of `mutator`  to construct `arch_param`, as we hope the mutable could work(build and forward) independently in our design.